sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -76,7 +80,9 @@ int pollSocketUntilEventOrTimeout(
   throw TransportException("Error while sending request. Platform does not support Poll()");
 #endif
 
-  struct pollfd poller;
+  struct pollfd poller
```

Comment:
why this change ?

Reply:
Not 100% sure, to be honest. My best guess is that some static analysis tool complained about it.

Reply:
Ok, found it - the linter is complaining that poller is uninitialized, and adding the {} initializer got formatted strangely.